test/verify/check-session.py

```diff
@@ -78,7 +78,7 @@ def cond():
         # Kill session from the inside
         b.switch_to_top()
         b.wait_visible("#content")
-        b.wait_text('#current-username', 'admin')
+        b.wait_text('#current-username', 'adminn')
```

Comment:
That surely isn't an intended change :-)

Reply:
sure. I've added it there to have the machine reserved :-) via `-s` option :-)